decidim-core/app/packs/src/decidim/form_filter.js

```diff
@@ -24,9 +24,9 @@ export default class FormFilterComponent {
     this._onPopState = this._onPopState.bind(this);
 
     if (window.Decidim.PopStateHandler) {
-      this.popStateSubmiter = false;
+      this.popStateSubmitter = false;
```

Comment:
this change seems to be contained to this file, but I suppose something could be peeking into the object from afar.

Reply:
Just 3 occurences for `popStateSubmiter` in Decidim develop branch (including node_modules). all are present in this file.